### PR TITLE
Fix: unescape escape sequences

### LINKED ISSUE #2225: bigquery whitespace characters need to be escaped in string and bytes literals
Looks like newlines are handled but the remaining characters are not escaped correctly.

**Fully reproducible code snippet**

```
In [22]: import sqlglot as sg

In [23]: sg.__version__
Out[23]: '18.0.0'

In [24]: sql = sg.parse_one("SELECT '\a\b\f\n\r\t\v'", read="bigquery").sql("bigquery")

In [25]: !bq shell <<< "$sql"
Welcome to BigQuery! (Type help for more information.)
ibis-gbq> Error in query string: Error processing job 'ibis-gbq:bqjob_r1fef3289cccddd3_0000018a987c16e2_1': Syntax error: Unclosed string literal at [1:8]
ibis-gbq> Goodbye.
```

**Official Documentation**

https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#escape_sequences

### PR DESCRIPTION
**Pull request by GeorgeSittas**
Fixes #2225

### CODE REVIEW

File: sqlglot/generator.py [hunk 1]
```diff
@@ -12,6 +12,18 @@
 
 logger = logging.getLogger("sqlglot")
 
+UNESCAPED_SEQUENCE_TABLE = str.maketrans(
```
Comment (tobymao) [inline sqlglot/generator.py:15]: this should be per dialect i think, everyone is different and configurable

File: sqlglot/generator.py [hunk 2]
```diff
@@ -320,7 +332,7 @@ class Generator:
     STRICT_STRING_CONCAT = False
     NORMALIZE_FUNCTIONS: bool | str = "upper"
     NULL_ORDERING = "nulls_are_small"
-    ESCAPE_LINE_BREAK = False
+    UNESCAPE_SEQUENCES = False
```
Comment (tobymao) [inline sqlglot/generator.py:335]: could probably remove this in favor of whether or not this mapping exists?

File: sqlglot/generator.py [hunk 3]
```diff
@@ -1594,8 +1606,8 @@ def literal_sql(self, expression: exp.Literal) -> str:
 
     def escape_str(self, text: str) -> str:
         text = text.replace(self.QUOTE_END, self._escaped_quote_end)
-        if self.ESCAPE_LINE_BREAK:
-            text = text.replace("\n", "\\n")
+        if self.UNESCAPE_SEQUENCES:
+            text = text.translate(UNESCAPED_SEQUENCE_TABLE)
         elif self.pretty:
             text = text.replace("\n", self.SENTINEL_LINE_BREAK)
```
Comment (tobymao) [inline sqlglot/generator.py:1612]: is this section correct?

File: sqlglot/generator.py (hunk 1 above)
Comment (GeorgeSittas) [reply sqlglot/generator.py:15]: sounds good with me, shouldn't be much of a change

File: sqlglot/generator.py (hunk 2 above)
Comment (GeorgeSittas) [reply sqlglot/generator.py:335]: sure

File: sqlglot/generator.py (hunk 3 above)
Comment (GeorgeSittas) [reply sqlglot/generator.py:1612]: The `SENTINEL_LINE_BREAK` one?

Comment (tobymao) [reply sqlglot/generator.py:1612]: yea when does this hit, just double check it

File: sqlglot/generator.py (hunk 2 above)
Comment (GeorgeSittas) [reply sqlglot/generator.py]: made these changes

File: sqlglot/generator.py (hunk 3 above)
Comment (GeorgeSittas) [reply sqlglot/generator.py:1601]: seems necessary based on https://github.com/tobymao/sqlglot/commit/fd4dbacb3

**Pull request merged**